examples/logrotate.service

```diff
@@ -24,9 +24,11 @@ PrivateDevices=true
 PrivateTmp=true
 ProtectClock=true
 ProtectControlGroups=true
+ProtectHostname=true
 ProtectKernelLogs=true
 ProtectKernelModules=true
 ProtectKernelTunables=true
 ProtectSystem=full
 RestrictNamespaces=true
 RestrictRealtime=true
+RestrictSUIDSGID=true
```

Comment:
I am not sure about this one.  Setting setuid/setgid bits on directories is quite common.  What is the motivation behind this change?

Reply:
Is it really common to set setuid/setgid bits inside of logrotate scripts?

Reply:
Who are you asking here?  Users of logrotate?  This is not the channel to reach them...

Note that these bits are not limited to scripts only.  They can be set by logrotate directly, for example: `createolddir 6770`

Reply:
OK, I understand now.
Converted into a comment why not using RestrictSUIDSGID